temporalio/workflow.py

```diff
@@ -981,6 +984,15 @@ def memo() -> Mapping[str, Any]:
     return _Runtime.current().workflow_memo()
 
 
+def is_workflow_failure_exception(err: BaseException) -> bool:
```

Comment:
```suggestion
def is_failure_exception(err: BaseException) -> bool:
```

I think we can remove the `workflow` qualifier because we removed it on the `@workflow.defn` decorator